pkgs/by-name/ai/aider/package.nix

```
@@ -0,0 +1,61 @@
+{ lib
+, python3
```

- #211340

Lets use `python3Packages` instead of `python.pkgs`

```suggestion
, python311Packages
```